doc/networking.md

```diff
@@ -327,14 +327,45 @@ possible to specify use of a random identifier ([ietf-ip][ietf-ip-yang] YANG and
 Both the link-local address (fe80::) and the global address (2001:)
 have changed type to *random*. 
 
+### IPv4 forwarding
+To be able to route (static or dynamic) on the interface it is
+required to enable forwarding. This setting controlls if packets
+received on this interface can be forwarded.
+   ```
+   admin@example:/config/> edit interface eth0
+   admin@example:/config/interface/eth0/> set ipv4 forwarding
+   admin@example:/config/interface/eth0/> leave
+   admin@example:/>
+   ```
+
+### IPv6 forwarding
+This flag behaves totally different than for IPv4. For IPv6 the
+ability to route between interfaces is always enabled, instead this
+flag controls if the interface will be in host/router mode.
+
+| Feature                                   | Forward enabled | Fordward disabled |
```

Comment:
Great that you added this section, and this table!  Just make sure to follow the table style, using `**Heading**` for column names.  Not all markdown formatters pretty print the heading.